Makefile

```diff
@@ -134,6 +134,7 @@ CI-test:
 
 OPENAPI_OUTPUT_DIR=${PWD}/tmp/openapi-client
 
+.PHONY: nethsm-api.yaml
```

Comment:
Why phony?  I think this makes it harder to test local changes because they would be overwritten every time the client is generated.  `--always-make` could be used instead to refresh the file.

Reply:
`make nethsm-client` already uses phony, so I thought it would be the most user-friendly solution.

Reply:
The difference is that there is no `nethsm-client` file that would be generated by the target, so it has to be phony.

Reply:
That would also make it harder to test changes locally (or to modify the upstream scheme if necessary).